docs/handbook/image-file-formats.rst

```diff
@@ -111,7 +111,9 @@ additional frames when saving, the ``append_images`` parameter works with
 
 If present, the ``loop`` parameter can be used to set the number of times
 the GIF should loop, and the ``duration`` parameter can set the number of
-milliseconds between each frame.
+milliseconds between each frame. The ``duration`` parameter can be either
+an integer or a list of integer. Passing a list to ``duration`` parameter
```

Comment:
"list of integer" -> "list of integers"
"to ``duration`` parameter" -> "to the ``duration`` parameter"